core/app/modules/import-export/iterator.php

```diff
@@ -46,10 +46,22 @@ public function __construct( array $settings ) {
 
 		$server = new Server();
 
-		$server_write_permissions = $server->get_write_permissions();
-
-		if ( $server_write_permissions['warning'] ) {
-			throw new \Error( self::NO_WRITE_PERMISSIONS_KEY );
+		$paths_to_check = [
+			Server::KEY_PATH_WP_CONTENT_DIR,
+			Server::KEY_PATH_UPLOADS_DIR,
+			Server::KEY_PATH_ELEMENTOR_UPLOADS_DIR,
+		];
+
+		$permissions = $server->get_system_paths_permissions();
+
+		$permissions = array_filter( $permissions, function ( $key ) use ( $paths_to_check ) {
+			return array_key_exists( $key, $paths_to_check );
+		}, ARRAY_FILTER_USE_KEY  );
+
+		foreach ( $permissions as $permission ) {
+			if ( ! $permission['write'] ) {
+				throw new \Error( self::NO_WRITE_PERMISSIONS_KEY );
+			}
```

Comment:
Maybe throw an error with all problematic paths

Reply:
A fix for all the errors and exceptions is currently being made by our product team. I think for now it's good enough just to throw a general error.